fateme/challenge-6.js

```diff
@@ -1,15 +1,25 @@
 function pyramid(n = 5) {
-  for (let index = 1; index <= n; index++) {
-    const line = []
-    for (let j = 1; j <= n - index; j++) {
-      line.push(" ")
+  if (isTypeValid(n, "number")) {
```

Comment:
This approach is great but for more complicated types (which we will cover in the TypeScript course).

Reply:
I remove type checking at all, hope to know what you mean